projects/commudle-admin/src/app/components/sidebar-menu/sidebar-menu.component.html

```diff
@@ -90,7 +90,7 @@
     </nb-accordion>
   </div>
 
-  <div *ngIf="isPageAdsAdmin || isFeaturedCommunitiesAdmin || isBadgesAdmin">
+  <div *ngIf="isPageAdsAdmin || isFeaturedCommunitiesAdmin || isBadgesAdmin || isAssetsAdmin">
```

Comment:
Can we move this condition into the .ts file in a variable and then call it here?

Reply:
Need to discuss